README.md

```diff
@@ -404,6 +421,43 @@ See more on [this blog post](https://medium.com/@tomchentw/imagemin-lint-staged-
 }
 ```
 
+## Utility functions
+
+_Lint-staged_ offers certain utility functions to make it easier to integrate into your project:
+
+### `escape`
+
+A method to escape a command line argument (typically a filename) for safer usage. **This should always be used in functional tasks when the `--shell` option is used!**.
+
+```js
+const { escape } = require('lint-staged/utils')
+const filename = '/a directory with spaces!/file.js'
+
+console.log(escape(filename)) // '\/a\\ directory\\ with\\ spaces\!\/file\.js'
+```
+
+#### Notes:
+
+- Spaces are double-escaped with `\\`. This is because [execa](https://github.com/sindresorhus/execa) has special handling for splitting from spaces.
+- **The escape method currently does nothing when `process.platform === 'win32'`!** If you want to help make it work on Windows, please open let us know!
+
+### `gitAdd`
```

Comment:
I've been meaning to add it, and since I also added `escape` I figured I'd throw it in as well. Now that I remember, these both could use a simple regression test.